docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+
+However, if a newer version of the data structure is subsequently deployed in production, you may wish to "upgrade" the tracking scenario and tracking to the newer data structure and verify that your implementation rules still meet validation.
+
+The fields available for filling at this stage are:
+
+- A text field for the **scenario name**
+- A tag field to indicate which applications this scenario is designed to track.
+- A free text field for a **description** of the tracking scenario.
+
+![](images/ts-3.png)
+
+To create a tracking scenario, click the **'Save'** button. If you want to create multiple scenarios at once, such as different scenarios for mobile and web tracking, click the **'Save & add another'** button. You can cancel the creation process by clicking the **'Cancel'** button.
+
+After creating your first tracking scenario, a list of tracking scenarios will appear on the page.
+
+![](images/ts-4.png)
+
+Until published, the scenario status is set to "Draft" to represent that it is a work in progress. During this time, you may wish to share it with others in the business to get feedback or approval. To do this, select the link to the tracking scenario and click the "Share" button located at the top right of the page. Please note that others will need access to the Snowplow Console Data Structures section to view the scenario.
+
+* * *
+
+## Adding more details and editing a Tracking Scenario
+
+To add more information or modify a tracking scenario, select it from the appropriate list view in the "Data Structure and Tracking Scenarios" section. This will open a new page displaying the details of the selected tracking scenario.
+
+![](images/ts-5.png)
+
+To make changes, edit the information in each section, and then click the ‘Save and Update’ button to save your changes.
+
+
+### Implementation rules
+
+The Implementation Rules section is where you can specify how each property for this event should be populated.
+
+To add multiple implementation rules, click the **'+ Add implementation rules'** button in the Implementation Rules section. This will open a dialog view.
+
+The dialog displays the list of properties defined in the data structure associated with this scenario.
+
+![](images/ts-7.png)
+
+You can click on any of the properties shown in the list to access a new view inside the dialog box. Here, you will be able to configure the rule's Type of condition, Value, and Comments properties.
+
+![](images/ts-8.png)
+
+Click **'Add rule'** to save the rule, or click the **'Cancel'** button to discard changes.
+
+After configuring the rules, click **'Save and update rules'** to confirm the changes or click **'Close'** to discard them.
+
+You can edit or remove rules in the tracking detail view's rule list.
+
+![](images/ts-9.png)
```

Comment:
This screenshot needs to be updated as it is not reflective of the up-to-date ui version

Reply:
![Screenshot 2023-06-26 at 09 45 05](https://github.com/snowplow/documentation/assets/6108743/7006bb62-0a9c-452b-b748-962429131280)

Up to date version of the screenshot

Reply:
Updated